### PR TITLE
Add debug message in SendMessageContext.

### PR DESCRIPTION
**Pull request by sfzylad**
This PR implements the #714

### CODE REVIEW

File: chat.go [hunk 1]
```diff
@@ -206,6 +208,14 @@ func (api *Client) SendMessageContext(ctx context.Context, channelID string, opt
 		return "", "", "", err
 	}
 
+	reqBody, err := ioutil.ReadAll(req.Body)
```
Comment (kanata2) [inline chat.go]: I think it is better for debugging but it may affect performance by `iotuil.ReadAll`.
How about you use `if api.Debug() { ... }` ?

Comment (sfzylad) [reply chat.go]: It makes perfect sense. Change is on the way! 👍

Comment (sfzylad) [reply chat.go]: For the record, I realise the `api.Debugf()` checks if the debug is enabled but I think it's more readable if it's inside of the if. :)

**Review by kanata2 — APPROVED** [review]
Thank you :)

**Pull request merged**